docs/developer.md

```diff
@@ -11,22 +11,14 @@ your repository with the upstream openebs/maya master. Syncing ensures your repo
 ### 2. Clone fork to local storage
 
 Place openebs/maya' code on your `GOPATH` using the following cloning procedure.
-
-Define a local working directory:
-
-```sh
-working_dir=$GOPATH/src/github.com/openebs
-```
 Create your clone:
 
 ```sh
-mkdir -p $working_dir
-cd $working_dir
-git clone https://github.com/$user/maya.git 
+go get github.com/$user/maya.git 
```

Comment:
@benjaminmisell -- Thanks for looking into this! 

My first thought, i might be a bit outdated here.. does go support compiling this?  doesn't it expect that the forked repository (github.com/$user/maya) be actually under ( github.com/openebs/maya)

Reply:
I belive it will download the fork and summing it to the original as well to keep compatibility.

Reply:
```
kiran@kmaya:~/go/src/github.com/openebs$ go get github.com/kmova/maya.git
package github.com/kmova/maya.git: invalid version control suffix in github.com/ path
```

I also tried with *go get github.com/kmova/maya*, this downloaded the code into kmova/maya. The make is returning the following error:

```
kiran@kmaya:~/go/src/github.com/kmova/maya$ make apiserver
GOOS=linux go build ./cmd/apiserver
cmd/apiserver/commands.go:7:2: cannot find package "github.com/openebs/maya/cmd/apiserver/app/command" in any of:
	/home/kiran/go/src/github.com/kmova/maya/vendor/github.com/openebs/maya/cmd/apiserver/app/command (vendor tree)
	/usr/local/go/src/github.com/openebs/maya/cmd/apiserver/app/command (from $GOROOT)
	/home/kiran/go/src/github.com/openebs/maya/cmd/apiserver/app/command (from $GOPATH)
GNUmakefile:103: recipe for target 'apiserver' failed
make: *** [apiserver] Error 1
```

Reply:
Oh sorry my bad. You need to go get the original then cd into it and set the origin to the fork. Will ammend my pr when I have time

Reply:
Thank you @benjaminmisell